src/Authentication/Authentication/Properties/Resources.resx

```diff
@@ -199,10 +199,19 @@
   <data name="NotConnectedToGraphException" xml:space="preserve">
     <value>The cmdlet cannot run because Authentication is set to {0} and Connect-Graph was not called. Invoke 'Connect-Graph' or specify Authentication to be 'UserProvidedToken' and Provide a Token then retry</value>
```

Comment:
```suggestion
    <value>The cmdlet cannot run because Authentication is set to {0} and Connect-MgGraph was not called. Invoke 'Connect-MgGraph ' or specify Authentication to be 'UserProvidedToken' and Provide a Token then retry</value>
```
nit: Let's avoid using aliases where possible since they could change.